form/src/main/java/org/dhis2/form/ui/FieldViewModelFactoryImpl.kt

```diff
@@ -31,6 +32,7 @@ class FieldViewModelFactoryImpl(
     private val uiEventTypesProvider: UiEventTypesProvider,
     private val keyboardActionProvider: KeyboardActionProvider,
     private val legendValueProvider: LegendValueProvider,
+    private val autoCompleteProvider: AutoCompleteProvider? = null,
```

Comment:
I will make it mandatory and provide it always like keyboardProvider, legendProvider..., independently if later it is used in the specific implementation